tests/test_bytes.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import pytest
+
+from obstore import Bytes
+
+
+@pytest.mark.skip(reason="Not implemented *yet*")
+def test_decode() -> None:
+    py_bytes = b"asdf"
+    ry_bytes = Bytes(py_bytes)
+    assert ry_bytes.decode() == "asdf"
```

Comment:
Can we remove this from this PR, because it doesn't implement decoding?

Reply:
boom